hello_world.py

```diff
@@ -6,3 +6,14 @@ def hello_world():
 hello_world()
 
 
+#I'm hoping to see comments about this line only
+
+def stupid_program():
+ print 'dummy'
+
+def another():
+    print 'this has proper line space'
+
+def yet_more():
```

Comment:
Reporter: FLAKE8
Rule: 0111
Severity: INFO
File: hello_world.py L17
 

Missing function docstring

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>